pkgs/by-name/rs/rs-git-fsmonitor/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  makeWrapper,
+  watchman,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rs-git-fsmonitor";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "jgavris";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-+5nR+/09HmFk3mq2B8NTeBT50aBG85yXEdeO6BhStVw=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-WkqJSbtaJxaagJMsdFiVozi1SkrfxXyM9bdZeimwJag=";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  fixupPhase = ''
+    wrapProgram $out/bin/rs-git-fsmonitor --prefix PATH ":" "${lib.makeBinPath [ watchman ]}" ;
+  '';
+
+  meta = with lib; {
+    description = "Fast git core.fsmonitor hook written in Rust";
+    homepage = "https://github.com/jgavris/rs-git-fsmonitor";
+    license = licenses.mit;
+    maintainers = [ ];
```

Comment:
If this is coming back, can you volunteer to maintain it?

Reply:
Yes sure, how can I add myself?

Reply:
See [maintainers/README.md](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md).  You should add yourself to the maintainer list in one commit, then add the package in a second commit.

Reply:
Done!